src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityOperatorBase.cs

```diff
@@ -13,12 +13,19 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     /// <summary>
     /// The record type includes synthesized '==' and '!=' operators equivalent to operators declared as follows:
-    /// 
+    ///
+    /// For record class:
     /// public static bool operator==(R? r1, R? r2)
     ///      => (object) r1 == r2 || ((object)r1 != null &amp;&amp; r1.Equals(r2));
     /// public static bool operator !=(R? r1, R? r2)
     ///      => !(r1 == r2);
-    ///        
+    ///
+    /// For record struct:
+    /// public static bool operator==(R r1, R r2)
```

Comment:
Should be left and right based on https://github.com/dotnet/roslyn/pull/51973?

Reply:
It will. That'll get picked up next time the feature branch is refreshed with changes from main branch.